exercises/hangman/example.js

```diff
@@ -0,0 +1,63 @@
+export const GAME_STATUS = {
+  WIN: 'win',
+  LOSS: 'loss',
+  ONGOING: 'ongoing'
+}
+
+export class Hangman {
+  constructor(word) {
+    this.remaining_guesses = 9
+    this.status = GAME_STATUS.ONGOING
+    this.word = word
+    this.guesses = []
+    this.maskedWord = ''
+    for (var i = 0; i < this.word.length; i++) {
```

Comment:
Instead, perhaps use `'_'.repeat(this.word.length)`?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat